src/helpers/providers/ReactTranslate.res

```diff
@@ -19,25 +19,8 @@ let cryptoMuggleTranslations: translations = %raw(`
     {
       locale: "en",
       crypto: {
-        bluetext: "Always for sale",
-        ethereum: "ethereum based",
-        tokens: "tokens",
-        subHeading:
-          "Link your social profile to your address and showcase your giving.",
-        nft: "Non-Fungible Token",
-        whatIsANFT:
-          " is a way to ensure digital assets are unique and easily tradable on a blockchain.",
-        harbergerTax: "Harberger Tax"
       },
       muggle: {
-        bluetext: "Always",
-        ethereum: "raising funds for",
-        tokens: "",
-        subHeading: "Adopt an animal to start giving today. ",
-        nft: "Blockchain Animal",
-        whatIsANFT:
-          " is a digital animal that is completely unique and easily tradable on a blockchain.",
-        harbergerTax: "Monthly Pledge"
```

Comment:
No longer used. (the bottom 3 haven't been used for more than half a year I guess...)